File: .tmp_py4013/batch_08_dump/2884_modify_columns.js
// LeetCode 2884 - Modify Columns
// https://leetcode.com/problems/modify-columns/

/**
 * @param {object[]} employees
 * @return {object[]}
 */
var modifySalaryColumn = function(employees) {
    return employees.map((r) => {
        if (Array.isArray(r)) return [r[0], r[1] * 2];
        return { ...r, salary: r.salary * 2 };
    });
};

File: .tmp_rs4013/rust_check_12/2884_modify_columns.rs
struct Solution;
// LeetCode 2884 - Modify Columns
// https://leetcode.com/problems/modify-columns/

impl Solution {
    pub fn modify_salary_column(employees: Vec<Vec<i32>>) -> Vec<Vec<i32>> {
        employees
            .into_iter()
            .map(|mut r| {
                if let Some(last) = r.last_mut() {
                    *last *= 2;
                }
                r
            })
            .collect()
    }
}

fn main() {}
